dep/monoidal-containers/github.json

```diff
@@ -0,0 +1,7 @@
+{
+  "owner": "bgamari",
+  "repo": "monoidal-containers",
+  "branch": "master",
```

Comment:
Might want to use my PR on this instead since it has some more fixes.

Reply:
oh, interesting

We're just shaving another yak right now actually, I should change the branch perhaps on my actual update branch, rather than this hasty merge of that and the branch we're using on interos... and/or I should review everything that's gone on here, and pull in the other direction.

Reply:
This PR is all but merged by Ben: https://github.com/bgamari/monoidal-containers/pull/41